basis/Holmakefile

```diff
@@ -1,19 +1,17 @@
-INCLUDES = ../developers ../misc pure ../translator ../characteristic ../semantics ../translator/monadic
-OPTIONS = QUIT_ON_FAILURE
+INCLUDES = $(CAKEMLDIR)/developers $(CAKEMLDIR)/misc\
+					 $(CAKEMLDIR)/semantics $(CAKEMLDIR)/characteristic\
+					 $(CAKEMLDIR)/translator $(CAKEMLDIR)/translator/monadic\
+					 pure
 
-all: $(DEFAULT_TARGETS) README.md
+CLINE_OPTIONS = --qof
 
-THYFILES = $(patsubst %Script.sml,%Theory.uo,$(wildcard *.sml))
-TARGETS0 = $(patsubst %Theory.sml,,$(THYFILES))
-TARGETS = $(patsubst %.sml,%.uo,$(TARGETS0))
-
-all: $(TARGETS) basis_ffi.o README.md
+all: $(DEFAULT_TARGETS) README.md basis_ffi.o
 .PHONY: all
 
 README_SOURCES = $(wildcard *Script.sml) $(wildcard *Lib.sml) $(wildcard *Syntax.sml) $(wildcard *.c) dependency-order
 DIRS = $(wildcard */)
 README.md: $(CAKEMLDIR)/developers/readme_gen readmePrefix $(patsubst %,%readmePrefix,$(DIRS)) $(README_SOURCES)
```

Comment:
If you're going to `protect` the file in the command, you should `dprot` it in the dependencies

Reply:
Thanks for the suggestion. Is there any chance of getting Holmakefile includes or similar so the boilerplate lines for README.md generation can be written (and updated) in just one place?

Reply:
See HOL-Theorem-Prover/HOL#511